Source/ProgressionSystemRuntime/Public/ProgressionSystemRuntimeModule.h

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Modules/ModuleInterface.h"
+
+/** Define Progression System log category. */
+PROGRESSIONSYSTEMRUNTIME_API DECLARE_LOG_CATEGORY_EXTERN(LogProgressionSystem, Log, All);
```

Comment:
Please completely remove here 'DEFINE_LOG_CATEGORY` and `DECLARE_LOG_CATEGORY_EXTERN` in cpp as I see you never used `UE_LOG`. I'm not fan of `UE_LOG` as well because it's very hard to notice them. Instead, we try to use 'ensureMsgf' and 'checkf' depending on context.